flink-table/flink-table-runtime/src/main/java/org/apache/flink/table/runtime/functions/table/lookup/fullcache/LookupFullCache.java

```diff
@@ -57,7 +57,7 @@ public synchronized void open(CacheMetricGroup metricGroup) {
         }
         metricGroup.hitCounter(hitCounter);
         metricGroup.missCounter(new SimpleCounter()); // always zero
-        cacheLoader.open(metricGroup);
+        cacheLoader.initializeMetrics(metricGroup);
     }
 
     public synchronized void open(Configuration parameters) throws Exception {
```

Comment:
I agree that 2 `open` methods is a problem, not only because of naming, but because of ambiguity in the order of calling these methods, and non-compliance with the contract of `LookupCache#open`, which should fully open the cache in one call. So I decided to keep just one `open` method and pass `Configuration` in different method. As a drawback we now have redundant state in cache, but I think it's a good compromise in our circumstances.